### PR TITLE
[BEAM-547] Version should be accessed from pom file

### PR DESCRIPTION
**Pull request by sb2nov**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

R: @aaltay PTAL

### CODE REVIEW

**Comment by aaltay** [comment]
I had https://github.com/apache/beam/pull/855 earlier to do this. I do not remember the full details but could you take a look at that and see if we can scrap anything from that?

**Comment by sb2nov** [comment]
R: @aaltay PTAL again

File: sdks/python/apache_beam/__init__.py [hunk 1]
```diff
@@ -77,6 +77,9 @@
 from apache_beam import coders
 from apache_beam import io
 from apache_beam import typehints
+from apache_beam import version
 from apache_beam.pipeline import Pipeline
 from apache_beam.transforms import *
 # pylint: enable=wrong-import-position
+
+__version__ = version.get_version()
```
Comment (aaltay) [inline sdks/python/apache_beam/__init__.py]: Why do we need this change?

Comment (sb2nov) [inline sdks/python/apache_beam/__init__.py]: Most python libraries expose version at the root level so you can do something like

```
import xyz
xyz.__version__
```

That isn't true in python so I wanted to make that happen. Eg. https://github.com/pandas-dev/pandas/blob/master/pandas/__init__.py#L71

**Comment by aaltay** [comment]
LGTM.

**Pull request closed without merging**